util/util.go

```diff
@@ -387,7 +387,7 @@ var (
 func fileExistsAndNotADir(path string) bool {
 	file, err := os.Stat(path)
 
-	if file == nil || err != nil || os.IsNotExist(err) {
+	if file == nil || err != nil || errors.Is(err, os.ErrNotExist) {
```

Comment:
This check makes no sense. If the `err != nil` we already go to the return since they are OR return here so there is no point in checking for `ErrNotExist` when err must be nil.